### ISSUE TITLE
Un using Adjustable Gradient Clipping ?

### ISSUE THREAD

**Issue by hjzhu001** [Jongchan/tensorflow-vdsr#32]
No discovery  gradient clipping technique when training network In VDSR.py？
